.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,43 @@
+name: changelog entry
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-changelog-entry:
+    name: Check for changelog entry
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'skip news') != true
```

Comment:
Thoughts on renaming this to `skip changelog`? I don't think we're using the "news" terminology in many other places, it's mostly "changelog".

Reply:
label name is shorter which is possibly the only benefit

Reply:
Yeah, we don't really need to worry about label length. Personally I'd prefer `skip changelog`, but it's your call